### PR TITLE
Podpeople species tail wag fix; also lifebringer name fix

### PR DESCRIPTION
**Pull request by BlueWildrose**
## About The Pull Request

- Podpeople can wag tails now; they weren't able to before for some reason.
- Fixed the ghost role lifebringers having no name.

## Why It's Good For The Game
Tail wagging should have been a thing for a species that could have tails, but wasn't.

No more "the a lifebringer" people because name changing didn't function properly for them.

## Changelog
:cl:
fix: podpeople tail wagging
tweak: lifebringer ghost role fixes
/:cl:

### CODE REVIEW

File: code/game/objects/structures/ghost_role_spawners.dm [hunk 1]
```diff
@@ -20,20 +20,30 @@
 	assignedrole = "Lifebringer"
 
 /obj/effect/mob_spawn/human/seed_vault/special(mob/living/new_spawn)
```
Comment (Ghommie) [inline code/game/objects/structures/ghost_role_spawners.dm]: Remove this line now.

File: code/modules/mob/living/carbon/human/species_types/podpeople.dm [hunk 2]
```diff
@@ -3,7 +3,9 @@
 	name = "Anthromorphic Plant"
 	id = "pod"
 	default_color = "59CE00"
-	species_traits = list(MUTCOLORS,EYECOLOR)
+	species_traits = list(EYECOLOR,HAIR,FACEHAIR,LIPS,MUTCOLORS)
+	mutant_bodyparts = list("mam_tail", "mam_ears", "mam_body_markings", "mam_snouts", "taur", "legs")
+	default_features = list("mcolor" = "FFF","mcolor2" = "FFF","mcolor3" = "FFF", "mam_snouts" = "Husky", "mam_tail" = "Husky", "mam_ears" = "Husky", "mam_body_markings" = "Husky", "taur" = "None", "legs" = "Normal Legs")
```
Comment (Ghommie) [inline code/modules/mob/living/carbon/human/species_types/podpeople.dm]: I'm very very very very very very against this. Like, can't you just man up and leave the standard one looking like a walking broccoli? It's the iconical side-effect of replica pods after all.
One can always bug xenobiology for some mutation toxin or something.

File: code/modules/mob/living/carbon/human/species_types/podpeople.dm [hunk 3]
```diff
@@ -70,13 +72,38 @@
...
+		mutant_bodyparts -= "mam_tail"
+		mutant_bodyparts |= "mam_waggingtail"
+	H.update_body()
+
+/datum/species/pod/stop_wagging_tail(mob/living/carbon/human/H)
+	if("mam_waggingtail" in mutant_bodyparts)
+		mutant_bodyparts -= "mam_waggingtail"
+		mutant_bodyparts |= "mam_tail"
+	H.update_body()
+
```
Comment (Ghommie) [inline code/modules/mob/living/carbon/human/species_types/podpeople.dm]: Remember to move this to the subspecies after reverting the above snowflakes change.

**Review by Ghommie — CHANGES_REQUESTED** [review]

**Comment by BlueWildrose** [comment]
Very well, the changes are made.

**Review by Ghommie — APPROVED** [review]

**Pull request merged**